shopinvader_quotation/services/quotation.py

```diff
@@ -28,12 +27,8 @@ def _validator_get(self):
         return {}
 
     def _validator_search(self):
-        return {
-            "id": {"coerce": to_int},
-            "per_page": {"coerce": to_int, "nullable": True},
-            "page": {"coerce": to_int, "nullable": True},
-            "scope": {"type": "dict", "nullable": True},
-        }
+        validator = self._default_validator_search()
+        return validator
```

Comment:
this can be shorter?
```suggestion
        return self._default_validator_search()
```

Reply:
Indeed, done.